client/src/components/Footer.js

```diff
@@ -1,75 +1,115 @@
-    <Container>
-      <div className={isSmallScreen ? "d-flex justify-content-center" : ""}>
-        <Row
-          className={cn(
-            "justify-content-center justify-content-lg-between text-white",
-            isSmallScreen
-              ? "d-flex flex-column w-max-content"
-              : "align-items-center"
-          )}
-          noGutters
-        >
-          <div className={cn("d-flex", isSmallScreen ? "flex-column" : "")}>
-            <div className="d-flex">
-              <div>Contact us:</div>
-              <a href="mailto:rs19team@googlegroups.com">
-                <FontAwesomeIcon
-                  className="ml-2"
-                  color={"white"}
-                  icon={faEnvelopeSquare}
-                />
-              </a>
-            </div>
-            <div className={cn("d-flex", !isSmallScreen ? "ml-7" : "")}>
-              <div>Stay in touch:</div>
-              <a href="https://www.facebook.com/groups/reversim/">
-                <FontAwesomeIcon
-                  className="ml-2"
-                  color={"white"}
-                  icon={faFacebook}
-                />
-              </a>
-              <a href="https://twitter.com/reversim/">
-                <FontAwesomeIcon
-                  className="ml-2"
-                  color={"white"}
-                  icon={faTwitter}
-                />
-              </a>
-              <a href="https://groups.google.com/forum/#!forum/reversim-summit">
-                <FontAwesomeIcon
-                  className="ml-2"
-                  color={"white"}
-                  icon={faEnvelope}
-                />
-              </a>
-            </div>
-            <a
-              className={cn("text-white", !isSmallScreen ? "ml-12" : "")}
-              href="http://confcodeofconduct.com/"
-            >
-              Code of Conduct
-            </a>
-          </div>
-          <div className="d-flex align-items-center text-white">
-            All Rights Reserved © 2019
-          </div>
-        </Row>
-      </div>
-    </Container>
-  </footer>
+const GlobalStyle = createGlobalStyle`
+  ${reset}
+  /* other styles */
+`;
+
+const FooterStyle = styled.footer`
+  padding: 50px;
+  background-color: ${props => props.theme.bkgr};
+  font-family: Source Code Pro, monospace;
+`;
+
+const List = styled.ul`
+  display: flex;
+  felx-direction: row;
+  justify-content: space-between;
+
+  @media (max-width: 900px) {
+    flex-direction: column;
+  }
+`;
+
+const ListItem = styled.li`
+  display: flex;
+  felx-direction: row;
+  color: ${props => props.theme.color};
+`;
+
+const Link = styled.a`
+  color: ${props => props.theme.color};
+  font-size: 16px;
+  cursor: pointer;
+  &:hover {
+    text-decoration: none;
+    color: ${props => props.theme.color};
+  }
+
+  @media (max-width: 1200px) {
+    font-size: 14px;
+  }
+`;
+
+const theme = {
+  bkgr: '#3312bc',
+  color: 'white',
+};
+
+const Footer = () => (
+  <ThemeProvider theme={theme}>
```

Comment:
We should wrap the entire `App` with that and see that we get those props here 
Also lets move the `theme` const to its own file somewhere at the top under `styles` dir

Reply:
Wrapped, works super!

Reply:
I also moved the `theme` object to a separate file, `Theme.js` and placed it in the components folder.
I imported `theme` to `App.js`.